test/python/circuit/library/test_logical_expression_oracle.py

```diff
@@ -0,0 +1,77 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test dimacs oracle circuits."""
+
+import unittest
+
+from qiskit.test.base import QiskitTestCase
+from qiskit.circuit import QuantumCircuit
+from qiskit.circuit.library.logical_expression_oracle import LogicalExpressionOracle
+from qiskit.quantum_info import Operator
+
+EXPRESSION_TEST = "(~v1 | ~v2 | ~v3) & (v1 | ~v2 | v3) & \
```

Comment:
Are these the only allowed symbols? Or does it also work with "not and or"? How was it in the old Aqua class?